EncounterMe/EncounterMe.Android/CustomRenderer/CustomSliderRenderer.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Android.Content;
+using Android.Views;
+using EncounterMe.Droid.CustomRenderer;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ExportRenderer(typeof(EncounterMe.RatingSlider), typeof(CustomSliderRenderer))]
+namespace EncounterMe.Droid.CustomRenderer
+{
+    public class CustomSliderRenderer : ViewRenderer
+    {
+        public CustomSliderRenderer(Context context) : base(context)
+        {
+        }
+
+        protected override void OnElementChanged(ElementChangedEventArgs<Xamarin.Forms.View> e)
+        {
+            base.OnElementChanged(e);
+        }
+
+        public override bool OnTouchEvent(MotionEvent e)
+        {
+            var touchX = e.GetX();
+            var touchY = e.GetY();
+
+            var slider = (Element as RatingSlider);
+
+            int n = slider.Children.Count;
+            double s = slider.ColumnSpacing;
+            double w = (this.Width - (s * (n - 1))) / n;
+
+            System.Diagnostics.Debug.WriteLine($"{touchX}");
```

Comment:
Ar skiriasi kazkuom System.Diagnostics.Debug.WriteLine nuo Console.WriteLine ?
Jei prie kazkurio daugiau pliusu, tai galima viska su tuo padaryt

Reply:
Jei vs nėra nustatyta kitaip, tai šitą matysi tik debuginimo metu, po projekto sustabdymo jo turėtų nelikti